html/closeup.html

```diff
@@ -9,15 +9,18 @@
       href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@300&display=swap"
       rel="stylesheet"
     />
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="../css/styles.css" />
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <!-- Script for tile mouseover and mouseout effects -->
```

Comment:
Either add this comment in each file or remove it (or if the code can be reused then just include it in the one place).